hero.py

```diff
@@ -0,0 +1,93 @@
+class Hero:
+    _hp: int
+    _damage: int
+    _level: int
+    _coins: int
```

Comment:
nice, but this is not java, you dont need to define the fields as class variables